virttest/utils_test/libvirt.py

```diff
@@ -3531,3 +3531,22 @@ def customize_libvirt_config(params,
                                   status_error='no')
 
     return obj_conf
+
+
+def check_qemu_cmd_line(content, err_ignore=False):
+    """
+    Check the specified content in the qemu command line
+    :param content: the desired string to search
+    :param err_ignore: True to return False when fail
+                       False to raise exception when fail
+    :return: True if exist, False otherwise
+    """
+    cmd = 'ps -ef|grep qemu|grep -v grep'
```

Comment:
you can use  command "pgrep -a qemu"  directly.

Reply:
Updated, thanks